contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -1203,10 +1309,10 @@ POP_WARNINGS
     assert(m_state != nullptr); // always set in constructor
     _erro("Some problems at accept: " << e.message() << ", connections_count = " << m_state->sock_count);
     misc_utils::sleep_no_w(100);
-    new_connection_.reset(new connection<t_protocol_handler>(io_service_, m_state, m_connection_type, new_connection_->get_ssl_support()));
-    acceptor_.async_accept(new_connection_->socket(),
-      boost::bind(&boosted_tcp_server<t_protocol_handler>::handle_accept, this,
-      boost::asio::placeholders::error));
+    (*current_new_connection).reset(new connection<t_protocol_handler>(io_service_, m_state, m_connection_type, (*current_new_connection)->get_ssl_support()));
```

Comment:
Do you actually need a new_connection_ipv6 ? It looks like the same object can be used for both.

Reply:
If we require IPv4 and IPv6 use the same port then yes, I believe it could.  I don't think it's a good idea to have two async_accept (one for each protocol, in case they're bound differently) to be bound to the same new_connection object, but I may be mistaken.

Reply:
OK, two is fine then.